lib/twitter-ads/account.rb

```diff
@@ -77,7 +77,7 @@ def all(client, opts = {})
     #
     # @return [String] The object instance detail.
     def inspect
-      str = String.new("#<#{self.class.name}:0x#{object_id}")
+      str = +"#<#{self.class.name}:0x#{object_id}"
```

Comment:
Just a note for myself: https://ruby-doc.org/core-2.4.0/String.html#method-i-2B-40
https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html

Reply:
I believe this was part of the 'rubocop --auto-correct' changes. Good to see that this was expected